File: Source/OregonTilth.Models/DataTransferObjects/Workbook/Forms/TransplantProductionStandardTime/TransplantProductionStandardTimeSummaryDto.cs


using System.Collections.Generic;

namespace OregonTilth.Models.DataTransferObjects
{
    public class TransplantProductionStandardTimeSummaryDto : IHasTimeStudies
    {
        public int? TransplantProductionStandardTimeID { get; set; }
        public int WorkbookID { get; set; }
        public WorkbookSummaryDto Workbook { get; set; }
        public TransplantProductionLaborActivitySummaryDto TransplantProductionLaborActivity { get; set; }
        public TransplantProductionTrayTypeSummaryDto TransplantProductionTrayType { get; set; }
        public IEnumerable<TimeStudySummaryDto> TimeStudies { get; set; }
        public decimal? StandardTimePerUnit { get; set; }
    }
}

File: OregonTilth.Web/src/app/shared/models/forms/transplant-production-standard-times/transplant-production-standard-time-summary-dto.ts
import { TransplantProductionLaborActivityDto } from "../../generated/transplant-production-labor-activity-dto";
import { TransplantProductionTrayTypeDto } from "../../generated/transplant-production-tray-type-dto";
import { TimeStudySummaryDto } from "../time-studies/time-studies-upsert-dto";

export class TransplantProductionStandardTimeSummaryDto {
    TransplantProductionStandardTimeID: number;
    WorkbookID: number;
    TransplantProductionLaborActivityID: number;
    TransplantProductionLaborActivity: TransplantProductionLaborActivityDto;
    TransplantProductionTrayTypeID: number;
    TransplantProductionTrayType: TransplantProductionTrayTypeDto;
    StandardTimePerUnit: number;
    TimeStudies: TimeStudySummaryDto[]
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}
